design.py

```diff
@@ -1,16 +1,40 @@
 import sys
+import json
 #prompt a user to create a recipe
 #Ask for Ingredients
 #Ask for Instructions
 #Save/Store it into dictionary
 def create():
-    ask_recipe = input("Recipe Name: ")
-    ask_instruction = input("Instructions: ")
-    ask_create = input("Ingredients: ")  
+    ask_recipe = ''
+    ask_instruction = ''
+    ask_ingredients = ''
+    recipe_dict = dict()
+    #counter = str(1)
+    print("Create a recipe! Type 'exit' to leave")
+    for ask_recipe in range (1):
+        ask_recipe = input("Recipe Name: ")
+        recipe_dict.update(ask_recipe = 'Nothing Yet')
+        if ask_recipe == 'exit':
+            exit(1)
 
+    while ask_instruction != 'exit':
+        ask_instruction = input("Instructions: ")
+        if ask_instruction == 'exit':
+            while ask_ingredients != 'exit':
+                ask_ingredients = input("Ingredients: ")  
+                if ask_ingredients == 'exit':
+                    #recipe_dict.update(ask_recipe = ask_instruction)
+                    #recipe_dict.update(ask_recipe, ask_ingredients)
```

Comment:
when you finally finish collecting your string and two lists then you can create a recipe object like we talked about during the design phase. This object is just the collection of the string and the two lists it you had collected. We can talk more about how to create his object later. For now focus on collecting the information you need.